Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mhart/alpine-node:6
+FROM node:8
```

Comment:
Please use `node:8.6-alpine` for smaller images:
`node:8`= 269 MB
`node:8.6-alpine` = 23 MB
See https://hub.docker.com/r/library/node/tags/

Reply:
done